packages/xchain-thorchain/src/client.ts

```diff
@@ -51,12 +55,8 @@ class Client implements ThorchainClient, XChainClient {
 
   constructor({ network = 'testnet', phrase }: XChainClientParams) {
```

Comment:
How about to provide `clienUrl` for `test|mainnet` as optional parameters? 
- `testnetClientUrl: string`
- `mainnetClientUrl: string`